src/openzaak/components/zaken/tests/test_auth.py

```diff
@@ -1386,6 +1388,12 @@ class InternalZaaktypeScopeTests(JWTAuthMixin, APITestCase):
     @classmethod
     def setUpTestData(cls):
         cls.zaaktype = ZaakTypeFactory.create()
+
+        ServiceFactory.create(
+            api_root="https://externe.catalogus.nl/api/v1/",
+            api_type=APITypes.ztc,
+        )
+
```

Comment:
Why do many tests now require the `https://externe.catalogus.nl/api/v1/` service.

I cannot find the reason myself.

Reply:
Actually nvm, I was running locally with `--keepdb` and because these services were created in setUpTestData, they were persisted :facepalm: 

They are needed for the usage of e.g. ZaakFactory with an external Zaaktype. The factory will fail if there is no Service that matches the base URL of the external ZaakType

Reply:
you're right, it was broken due to the slug field, I've fixed it